File: Android/TUIKit/TUIRoomKit/tuiroomkit/src/main/java/com/tencent/cloud/tuikit/roomkit/ConferenceError.java
package com.tencent.cloud.tuikit.roomkit;

public enum ConferenceError {
    SUCCESS(0),
    FAILED(-1),
    CONFERENCE_ID_NOT_EXIST(-2100),
    CONFERENCE_ID_INVALID(-2105),
    CONFERENCE_ID_OCCUPIED(-2106),
    CONFERENCE_NAME_INVALID(-2107);

    int mValue;

    ConferenceError(int value) {
        mValue = value;
    }

    public static ConferenceError fromInt(int value) {
        for (ConferenceError error : ConferenceError.values()) {
            if (error.mValue == value) {
                return error;
            }
        }
        return FAILED;
    }
}

File: iOS/TUIKit/TUIRoomKit/Source/Model/ConferenceError.swift
//
//  ConferenceError.swift
//  TUIRoomKit
//
//  Created by janejntang on 2024/3/14.
//

import Foundation

@objc public enum ConferenceError: Int {
    case success = 0
    case failed = -1
    case conferenceIdNotExist = -2100
    case conferenceIdInvalid = -2105
    case conferenceIdOccupied = -2106
    case conferenceNameInvalid = -2107
}
